setup/create_python_env.bash

```diff
@@ -16,7 +16,7 @@ echo "Created Python virtual environment in $VIRTUALENV_FOLDER"
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 pushd $SCRIPT_DIR/../pyrobosim
 python -m pip install --upgrade pip
-pip install lark pytest pytest-dependency pytest-html wheel
+pip install catkin-pkg empy lark pytest pytest-dependency pytest-html wheel
```

Comment:
snuck this in because it helps resolve some weird colcon build issues

Reply:
What the heck, `catkin`? That's really weird

Reply:
Can you at least leave a comment with a link to where you found this?